flaskr_tests.py

```diff
@@ -62,14 +81,58 @@ def test_edit(self):
             follow_redirects=True)
         self.app.get('/entry=1', follow_redirects=True)
         rv = self.app.post(
-            '/entry=1', 
-            data = dict(title='Bye',
-                        text='Nothing is allowed here'),
+            '/entry=1',
+            data=dict(title='Bye',
+                      text='Nothing is allowed here'),
             follow_redirects=True)
         assert b'HTML' not in rv.data
         assert b'Entry was successfuly edited' in rv.data
         assert b'Bye' in rv.data
 
+    def test_delete(self):
+        """Test entry delete."""
+        self.login('admin', 'default')
```

Comment:
I've added two more classes, but not really sure that I've done everything correctly, please have a look and let me know if I violated some basic OOP principles :see_no_evil: